go.mod

```diff
@@ -123,7 +124,9 @@ require (
 )
 
 replace (
-	github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.3
+	github.com/cosmos/cosmos-sdk => github.com/notional-labs/cosmos-sdk v0.45.1-allfast
+	github.com/cosmos/iavl => github.com/osmosis-labs/iavl v0.17.3-osmo-v5.0.20220321022426-8371754f3299
+	github.com/cosmwasm/wasmd => github.com/notional-labs/wasmd v0.21.1-0.20220319082325-607293cd0bfe
```

Comment:
Why are we moving off mainline wasmd?

The whole point of lupercalia was to move _to_ mainline wasmd and collect security patches along the way.

Just trying to understand reasoning here :)

Reply:
these forks should be removed from this pull request

Reply:
ah ok so the reasoning is interchain accounts

no fork no ica

wdyt